### PR TITLE
Add container images


### PR DESCRIPTION
**Pull request by evrardjp**
This allow us to build new images to replace hyperkube.

### CODE REVIEW

**Comment by evrardjp** [comment]
> Please squash all the commit and add comment that it is related to Kubernetes v1.18 upgrade

Will squash when we are sure we don't need one of the images I just removed here, like the kubernetes client.

We have so many different things to deal with kubectl, a little standardization wouldn't hurt.

**Comment by mjura** [comment]
I'm wondering if we should remove https://github.com/SUSE/caasp-container-images/tree/master/caasp-hyperkube-image in this patch or next one

**Comment by evrardjp** [comment]
We shouldn't remove it immediately, as I expect we might have to maintain hyperkube a little bit further (it will take some time for which people will not have migrated off it, for which we might still have maintenance patches, I guess?)

In any case, I thought it was orthogonal to the new image.

**Comment by evrardjp** [comment]
This _builds_ in my SLE 15 SP1 environment with latest kubernetes 1.18 from @mjura .

File: caasp-kube-apiserver-image/caasp-kube-apiserver-image.kiwi [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/kube-apiserver"
+        tag="v%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <subcommand execute="/bin/sh"/>
```
Comment (mjura) [inline caasp-kube-apiserver-image/caasp-kube-apiserver-image.kiwi]:
```suggestion
        <entrypoint execute="kube-apiserver"/>
        <subcommand clear="true"/>
```

File: caasp-kube-controller-manager-image/caasp-kube-controller-manager-image.kiwi [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/kube-controller-manager"
+        tag="v%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <subcommand execute="/bin/sh"/>
```
Comment (mjura) [inline caasp-kube-controller-manager-image/caasp-kube-controller-manager-image.kiwi]:
```suggestion
        <entrypoint execute="kube-controller-manager"/>
        <subcommand clear="true"/>
```

File: caasp-kube-proxy-image/caasp-kube-proxy-image.kiwi [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/kube-proxy"
+        tag="v%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <subcommand execute="/bin/sh"/>
```
Comment (mjura) [inline caasp-kube-proxy-image/caasp-kube-proxy-image.kiwi]:
```suggestion
        <entrypoint execute="kube-proxy"/>
        <subcommand clear="true"/>
```

File: caasp-kube-scheduler-image/caasp-kube-scheduler-image.kiwi [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/kube-scheduler"
+        tag="v%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <subcommand execute="/bin/sh"/>
```
Comment (mjura) [inline caasp-kube-scheduler-image/caasp-kube-scheduler-image.kiwi]:
```suggestion
        <entrypoint execute="kube-scheduler"/>
        <subcommand clear="true"/>
```

**Pull request merged**